firefox-ios/Client/Frontend/Settings/WebsiteDataManagement/WebsiteDataManagementViewController.swift

```diff
@@ -68,6 +68,8 @@ class WebsiteDataManagementViewController: UIViewController,
         tableView.isEditing = true
         tableView.allowsMultipleSelectionDuringEditing = true
         tableView.allowsSelectionDuringEditing = true
+        tableView.register(ThemedTableViewCell.self, forCellReuseIdentifier: ThemedTableViewCell.cellIdentifier)
+        tableView.register(ThemedTableViewCell.self, forCellReuseIdentifier: "showMoreCell")
```

Comment:
Nit: can we pull this string literal out into a shared property? Maybe something like a `static let …` on WebsiteDataManagementVC (if that's the only VC we're going to use it with).